### ISSUE TITLE
Lexer incorrectly parses ```<=``` operator

### ISSUE THREAD

**Issue by santi224m** [Fanaticaj/CPSC-323-Compiler#1]
When the compiler is run on the default ```RAT24S.source``` file, it parses the ```<=``` operator as a ```<``` and a ```=``` token instead of a ```<=``` token.

Line in source code:

```txt
while (low <= high )
```

Token output:

```txt
Token(type='keyword', value='while'), Token(type='separator', value='('), Token(type='identifier', value='low'), Token(type='operator', value='<'), Token(type='operator', value='='), Token(type='identifier', value='high'), Token(type='separator', value=')')
```

**Comment by santi224m** [comment]
The same thing happens for all operators of length 2.

```bash
================================  Source Code  =================================
+	-	*	/
==	!=	>	<
<=	>=	=

================================================================================ 

Token count: 14

[Token(type='operator', value='+'), Token(type='operator', value='-'), Token(type='operator', value='*'), Token(type='operator', value='/'), Token(type='operator', value='='), Token(type='operator', value='='), Token(type='operator', value='='), Token(type='operator', value='>'), Token(type='operator', value='<'), Token(type='operator', value='<'), Token(type='operator', value='='), Token(type='operator', value='>'), Token(type='operator', value='='), Token(type='operator', value='=')]
```

**Comment by santi224m** [comment]
Fixed by commit 0d745d6b0f70414c7d692cfd4872bfe1cc124b09.
Just needed to separate source code using separators
